src/common/serviceworker/CommonMailWorker.ts

```diff
@@ -0,0 +1,41 @@
+// @bundleInto:common
```

Comment:
1. the name is very misleading. Should `CommonMailUtils` be `MailUtils` (or `SharedMailUtils`) and this one be `CommonMailUtils`?
2. Why is it in service worker directory of all places? Why is it not in `common/src/api/common` where "common" (as in: common between main thread & worker) code resides?